### ISSUE TITLE
Please consider screen-reader specific how to use the editor information

### ISSUE THREAD

**Issue by brichwin** [tinymce/tinymce-docs#1541]
**Documentation/Product Version**
_TinyMCE 4 Documentation_ OR _TinyMCE 5 Documentation_

**Page to be updated (if applicable)**
https://www.tiny.cloud/docs/advanced/accessibility/

**Is your feature request related to a problem? Please describe.**
I've asked several screen-reader dependent users to insert an image or otherwise complete a task that requires using features only in a toolbar. They were successful in finding the keystroke to access the toolbars, but the Mac OS VoiceOver users each failed to successfully navigate the toolbars as they didn't understand how to interact and stop interacting with them. Some editor functions are only available in the toolbars.

**Describe the solution you'd like**

- Consider describing the differences between the Menus and the Toolbars. Why would they want one over the other?
- Consider specific help for using the toolbars with these screen-readers: VoiceOver (Mac), VoiceOver (iOS), JAWS, and NVDA. These are the most commonly used screen-reading softwares used by students in higher education.

**Describe alternatives you've considered**
We've considered writing our own documentation and providing it for students, however it is difficult to know where to put it so users would find it. Also, it's difficult to keep documentation maintained for every 3rd party tool used in a university environment.

**Additional context**
Thanks for the consideration!
